consensus/polybft/state_sync_manager.go

```diff
@@ -86,9 +86,10 @@ type topic interface {
 // NewStateSyncManager creates a new instance of state sync manager
 func NewStateSyncManager(logger hclog.Logger, state *State, config *stateSyncConfig) (*stateSyncManager, error) {
 	s := &stateSyncManager{
-		logger: logger.Named("state-sync-manager"),
-		state:  state,
-		config: config,
+		logger:  logger.Named("state-sync-manager"),
+		state:   state,
+		config:  config,
+		closeCh: make(chan struct{}),
```

Comment:
Could you check are any linters that can catch that kind of patterns?

Reply:
I have searched for it, but unfortunately haven't managed to find such.

Reply:
ruleguard? check type for channel fields, collect fields names, check for a constructor by pattern `n(N)ewTypeName` and check initialization.